scripts/Interface/reduction_gui/reduction/diffraction/diffraction_adv_setup_script.py

```diff
@@ -195,7 +195,7 @@ def to_xml(self):
                 # special map for bool type
                 value = '1' if value else '0'
             else:
-                value = ";".join(self.cache_dir) if keyname == "CacheDir" else str(value)
+                value = self.cache_dir.replace(",", ";") if keyname == "CacheDir" else str(value)
```

Comment:
To have better consistency between things, only use commas
```suggestion
                value = self.cache_dir if keyname == "CacheDir" else str(value)
```